04/app.js

```diff
@@ -9,11 +10,34 @@ class App extends React.Component {
         filesList: [],
     }
 
+    getFile = (e) => {
```

Comment:
Można było też odbieranie plików zrobić w `File`, a tutaj przekazywać już same dane.
To nie oznacza, że tak jest źle - po prostu alternatywne rozwiązanie :)